<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddQuoteDialog</class>
 <widget class="QDialog" name="AddQuoteDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Devis nÂ°XX</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../icons.qrc">
    <normaloff>:/icons/FactDev</normaloff>:/icons/FactDev</iconset>
  </property>
  <layout class="QGridLayout" name="_2">
   <item row="9" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Sauvegarder</string>
       </property>
       <property name="icon">
        <iconset resource="../../icons.qrc">
         <normaloff>:/icons/img/ok_valid.png</normaloff>:/icons/img/ok_valid.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Annuler</string>
       </property>
       <property name="icon">
        <iconset resource="../../icons.qrc">
         <normaloff>:/icons/img/cancel.png</normaloff>:/icons/img/cancel.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Date</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Titre</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="CheckUntilField" name="leQuoteTitle"/>
   </item>
   <item row="3" column="1">
    <widget class="QTextEdit" name="leDescription">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>80</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDateEdit" name="dateEditQuote">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="calendarPopup">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QWidget" name="wdgContributories" native="true"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Prestations</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::Widgets::CheckUntilField</class>
   <extends>QLineEdit</extends>
   <header>gui/widgets/checkfields/checkuntilfield.h</header>
   <slots>
    <slot>fieldTextChanged(QString)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnSave</sender>
   <signal>released()</signal>
   <receiver>AddQuoteDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>639</x>
     <y>340</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>released()</signal>
   <receiver>AddQuoteDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>725</x>
     <y>340</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leQuoteTitle</sender>
   <signal>textChanged(QString)</signal>
   <receiver>AddQuoteDialog</receiver>
   <slot>updateBtn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>-67</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leQuoteTitle</sender>
   <signal>textChanged(QString)</signal>
   <receiver>leQuoteTitle</receiver>
   <slot>fieldTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>406</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>contributoriesChanged()</signal>
  <slot>updateBtn()</slot>
 </slots>
</ui>
